Jenkinsfile

```diff
@@ -90,50 +103,24 @@ timestamps {
               archiveArtifacts 'coverage-report/lcov-report/index.html'
             }
           }
-        }
-
-        stage('Sonar') {
-          onPR {
-            sh """
-              yarn sonar-scanner -- \
-              -Dsonar.analysis.mode=preview \
-              -Dsonar.host.url=$SONARQUBE_URL
-            """
-          }
-
-          onMaster {
+          stage('Sonar') {
             sh "yarn sonar-scanner -- -Dsonar.host.url=$SONARQUBE_URL"
           }
-        }
-
-        stage('Package application (Docker)') {
```

Comment:
I think we should still build Docker images and run integration tests on master.

This is also needed (by default) to tag docker images with latest tag so that we can pull these from Artifactory locally.

Reply:
Agree, they should be run on master as well.

Reply:
right, oops